interview-questions.md

```diff
@@ -14,44 +14,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: git is software that lets people save programming work. Github is the user interface platform.
 
-Researched answer:
+Researched answer:Git is a free and open source DevOps software tool that lets you manage and track your source code history. Github is a cloud-based hosting service that lets you manage Git repositories.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: relational operators
```

Comment:
When answering interview questions, you should expand using definitions and examples when possible.  One-word or two-word answers in an interview setting will not show your understanding and knowledge, and will likely come off as uninterested.